### PR TITLE
More on docs and solved bug in read_bsa

### PR DESCRIPTION
**Pull request by brk00**
Rebased from #23

### CODE REVIEW

**Comment by brk00** [comment]
Travis will fail because DSP isn't exporting FilterCoefficients. See JuliaDSP/DSP.jl#103

**Comment by codles** [comment]
Nice PR on DSP.jl.
I changed the CI tools to checkout the master branch till its released

**Comment by brk00** [comment]
I believe it's ready to merge

**Comment by brk00** [comment]
Rebased and squashed commits

**Comment by codles** [comment]
The error `ERROR: error in method definition: function Winston.oplot must be explicitly imported to be extended` from [travis](https://travis-ci.org/codles/EEG.jl/builds/60690137) might be fixed by adding `import Winston.oplot`

**Comment by brk00** [comment]
Tests are now passing :sunny:

**Comment by brk00** [comment]
Created a .jl file for each file type. There are defined IO related functions for each of these types.

File: src/EEG.jl [hunk 1]
```diff
@@ -126,17 +127,23 @@ export # Helper functions
 include("miscellaneous/helper.jl")
 
 # File type reading and writing
-include("read_write/read.jl")
-include("read_write/write.jl")
-include("read_write/biosemi.jl")
-include("read_write/besa.jl")
+include("read_write/io.jl")
```
Comment (codles) [inline src/EEG.jl]: This file does not exist and is causing the builds to fail

**Comment by codles** [comment]
Thanks mate, once this builds I'll merge it.

**Comment by codles** [comment]
Thanks for all your work. The code is starting to look much cleaner already

**Pull request merged**